modules/planning/lattice/trajectory_generation/trajectory_evaluator.cc

```diff
@@ -173,7 +174,7 @@ double TrajectoryEvaluator::Evaluate(
   std::vector<double> s_values;
   for (double s = 0.0; s < evaluation_horizon;
        s += FLAGS_trajectory_space_resolution) {
-    s_values.push_back(s);
+    s_values.emplace_back(s);
```

Comment:
this is not necessary. push or emplace here have no difference.

Reply:
For native data types, the “emplace_back” or "emplace" function doesn't improve efficiency over the "push_back" or "push" fucntion; but for non-native data types, the formmer doesn't invoke an extra constructor and thus the efficiency is enhanced. Whether for native or non-native data types, the C++11 standard recommends using the "emplace_back" function. Therefore, I think my change is no problem.